lib/spinach/reporter/reporting.rb

```diff
@@ -126,7 +126,7 @@ def summarized_error(error)
       #  The coplete error report
       #
       def full_error(error)
-        feature, scenario, step, exception = error
+        _, _, step, exception = error
```

Comment:
`_feature` and `_scenario` are probably better names for unused variables?

Reply:
they were throwing wanings due to being assigned and never used.  They aren't actually part of the test, which is why I removed them.  Makes me wonder if maybe the error method should be returning something different?  Or there's something else we should be testing that we aren't?

I'm ok with leaving them in, or changing the tests.

Reply:
You may be referring to rubocop's Lint/UselessAssignment…  If so, prepending a `_` at the beginning of the variable name silents this warning.  Here, we call a method which return an array of 4 elements, and we are only interested in the 3rd and 4th ones, so yes, we need some "placeholders" to store the 1st and 2nd.

Using just `_` as a name is possible but is quite obscure.  For the sake of readability, it may make sense to keep the original names.

This is clearly just a matter of style :wink:

Reply:
agreed that it's not as clear.  I'll back this out.